sdk/core/core-tracing/README.md

```diff
@@ -16,40 +16,58 @@ npm install @azure/core-tracing
-tracer.addSpanProcessor(new SimpleSpanProcessor(exporter));
+### OpenTelemetry Compatibility
+
+Both the Azure SDK and Microsoft's [Application Insights](https://www.npmjs.com/package/applicationinsights) use [OpenTelemetry](https://opentelemetry.io/) to support tracing. Specifically, we depend on the [@opentelemetry/api](https://www.npmjs.com/package/@opentelemetry/api) npm package.
+
+As OpenTelemetry iterated on their API towards their 1.0 GA release, both libraries were updated to match.
 
-setTracer(tracer);
+Some incompatibility between the two libraries is due to mismatches between the OpenTelemetry versions used in either `@azure/core-tracing` or `applicationinsights` when the two are used side-by-side. For folks who are using both libraries in the same application, we recommend using the same version of OpenTelemetry for both libraries by upgrading to their latest versions.
 
-const rootSpan = tracer.startSpan("root");
-const context = opentelemetry.setSpan(opentelemetry.context.active(), rootSpan);
+> Please note that we do not foresee any future compatibility concerns now that OpenTelemetry 1.0.0 has been released and the API considered stable.
+
+#### Compatibility Matrix
+
+| Core Tracing     | Application Insights | @opentelemetry/api |
+| ---------------- | -------------------- | ------------------ |
+| 1.0.0-preview.10 |                      | 0.10.2             |
+| 1.0.0-preview.11 | 2.1.0                | 1.0.0-rc.0         |
+| 1.0.0-preview.11 | 2.1.1                | 1.0.0-rc.0         |
+| 1.0.0-preview.11 | 2.1.2                | 1.0.0-rc.0         |
+| 1.0.0-preview.11 | 2.1.3                | 1.0.0-rc.0         |
+| 1.0.0-preview.12 | ^2.1.4               | ^1.0.0             |
+| 1.0.0-preview.13 | ^2.1.4               | ^1.0.0             |
+
+Please see the [troubleshooting](#troubleshooting) section for additional information about handling compatibility errors.
+
+## Examples
 
-// Call some client library methods and pass rootSpan via tracingOptions.
+### Example 1 - Enabling tracing using OpenTelemetry
 
-rootSpan.end();
-exporter.shutdown();
+```ts
+const opentelemetry = require("@opentelemetry/api");
+const { SimpleSpanProcessor, ConsoleSpanExporter } = require("@opentelemetry/tracing");
+
+const provider = new NodeTracerProvider();
+provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+provider.register();
+
+// Call some client library methods using automatic span propagation.
 ```
 
-### Example 2 - Passing current Context to library operations
+### Example 2 - Manual Span Propagation using OpenTelemetry
 
-```js
-// Given a BlobClient from @azure/storage-blob
+```ts
+// Given a BlobClient from @azure/storage-blob, a given context, and a global tracer provider as per the previous example.
+// The context is passed to the client library as a tracingContext option.
 const result = await blobClient.download(undefined, undefined, {
   tracingOptions: {
     tracingContext: context
```

Comment:
We dont have to talk about how to create this context?

Reply:
I addressed or replied to all the comments. Thanks for taking a look! (will be resolving anything I addressed)